scripts/build/ci-build/Dockerfile

```diff
@@ -71,10 +71,10 @@ RUN apt-get update   && \
 # base image to crossbuild grafana
 FROM ubuntu:14.04
 
-ENV GOVERSION=1.13.1 \
+ENV GOVERSION=1.13.4 \
```

Comment:
Due to changes in this file you'll need to update/increment buildcontainer version and push to docker hub, see https://github.com/grafana/grafana/blob/master/scripts/build/ci-build/build-deploy.sh circleci. When that's done you want to update all references to grafana/build-container in the repository.